help/destinations/catalog/crm/salesforce.md

```diff
@@ -10,7 +10,7 @@ exl-id: bd9cb656-d742-4a18-97a2-546d4056d093
 
 [!DNL Salesforce CRM](https://www.salesforce.com/) is a popular Customer Relationship Management (CRM) platform.
 
-This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [Salesforce REST API](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/dome_composite_upsert_example.htm?q=contacts), which allows you to update identities within a segment into [!DNL Salesforce CRM].
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [Salesforce REST API](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/resources_composite_sobjects_collections_update.htm), which allows you to update [Contact](https://developer.salesforce.com/docs/atlas.en-us.object_reference.meta/object_reference/sforce_api_objects_contact.htm) and [Lead](https://developer.salesforce.com/docs/atlas.en-us.object_reference.meta/object_reference/sforce_api_objects_lead.htm) objects within a segment into [!DNL Salesforce CRM].
 
```

Comment:
Let's add a note here with more quick information about leads and contacts. Taken from a [thread](https://launch-developers.slack.com/archives/C02Q3542CUU/p1663335024130219?thread_ts=1661880772.863439&cid=C02Q3542CUU) with Vishal: 

>[!NOTE]
>
>Salesforce leads vs. contacts: A lead is the name of a person or company who may (or may not) be interested in the products or services you sell. A contact is an individual with whom one of your reps has established a relationship and is a potential customer.
In Salesforce, a lead is an individual or company that might be interested in what you sell.
A contact is a lead that has been qualified as a potential purchaser.
The composite API supports both leads and contacts and user can select the type update attributes and segment data for these two.

You can wordsmith the note a bit, but please keep the general idea.